server-side-rendering/src/routes/+page.server.ts

```diff
@@ -0,0 +1,21 @@
+import { SESSION_COOKIE, createAppwriteClient } from '$lib/server/appwrite.js';
+import { redirect } from '@sveltejs/kit';
+
+export async function load({ locals }) {
+	if (!locals.user) throw redirect(301, '/signin');
+
+	return {
+		user: locals.user
+	};
+}
+
+export const actions = {
+	default: async (event) => {
```

Comment:
Is this typed automatically? If not, lets type those

Reply:
It's typed automatically.